backend/minNow/settings.py

```
@@ -315,3 +315,27 @@
+ES_CLIENT = None
+try:
+    from elasticsearch import Elasticsearch
+    
+    if ES_API_KEY:
+        ES_CLIENT = Elasticsearch([ES_HOST], api_key=ES_API_KEY)
+    else:
+        ES_CLIENT = Elasticsearch([ES_HOST])
+    
+    # Test connection (optional - don't fail if ES is down)
+    if ES_CLIENT and ES_CLIENT.ping():
+        log.info(f"✓ Connected to Elasticsearch at {ES_HOST}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)